### ISSUE TITLE
Option not available to change the BIOS type

### ISSUE THREAD

**Issue by harshit0921** [nutanix/terraform-provider-nutanix#163]
**Terraform Version**
Terraform v0.12.24

**Affected Resource(s)**
nutanix_virtual_machine
Terraform Configuration Files

Create VM with a disk that references an image:

```
resource "nutanix_virtual_machine" "MyVM_TF" {
  for_each         	= toset(var.vm-name)
  name             	= each.value
  description          = "Created by ${var.nutanix-user}"
  cluster_uuid         = var.cluster-uuid
  num_vcpus_per_socket = 1
  num_sockets          = 4
  memory_size_mib      = 12288

  nic_list {
    subnet_uuid = var.vm-subnet-uuid
  }

  disk_list {
  data_source_reference = {
   kind = "image"
   uuid = var.nutanix-image-uuid
    }
   }

}
```
  
**Debug Output**
None

**Panic Output**
None

**Expected Behavior**
The BIOS type is selected as UEFI

**Actual Behavior**
The BIOS type is selected as Legacy BIOS

**Steps to Reproduce**
Create a VM with an image
Check the properties of the VM

When creating a VM through terraform, I am unable to select the BIOS as UEFI. The created VM always has legacy BIOS selected.
Even in the documentation, I wasn't able to find any option to select the BIOS type.

**Comment by harshit0921** [comment]
Hi @marinsalinas can you please look into this. Thanks

**Comment by MaxPeal** [comment]
Please also don't forget the settings for secure boot

**Comment by marinsalinas** [comment]
@harshit0921 @MaxPeal added a fix on #217 can you take a look?

**Comment by MaxPeal** [comment]
Not at the moment as I have no dedicated test system
